corehq/apps/users/views/mobile/users.py

```diff
@@ -1173,6 +1177,7 @@ def page_url(self):
         return reverse(self.urlname, args=self.args, kwargs=self.kwargs)
 
 
+@method_decorator([FILTERED_BULK_USER_DOWNLOAD.required_decorator()], name='dispatch')
```

Comment:
possibly an oversight earlier. The [link to the view is behind the feature flag](https://github.com/dimagi/commcare-hq/blob/c5e48c8cf8661f4c6cddec150f57bab415ed52ba/corehq/apps/users/views/mobile/users.py#L675) but the view itself was not checking for it.
Similarly for count_users, which is called only for this view

Reply:
Nice catch.